### ISSUE TITLE
Units of measurement error?

### ISSUE THREAD

**Issue by jx5c** [OpenMap-java/openmap#26]
In **OMDistance.java**, the statement: 

float latDist = (float) g2.getLatitude() - (float) g1.getLatitude();

latDist should in the unit of radians, but **getLatitude**() appears to return degree values.

**Comment by dfdietrick** [comment]
yeah, I think that should probably be changed.  It's only doing that for LINETYPE_STRAIGHT, but it should return the same value as it does for LINETYPE_GREATCIRCLE, since calculating the distance of straight lines doesn't really make sense.  Thanks for the heads-up.

- Don
